### PR TITLE
Add batch fargate templates

### PR DESCRIPTION
**Pull request by delagoya**
*Issue #, if available:* Resolves issues #2, #19, #20 

*Description of changes:*

Added Fargate tutorial and infrastructure examples. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: fargate/cloudformation/batch-fargate-new-vpc.yaml [hunk 1]
```diff
@@ -0,0 +1,357 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (mhuguesaws) [inline fargate/cloudformation/batch-fargate-new-vpc.yaml:1]: I believe core infrastructure is generic and should belong into a `core_infra/ec2-fargate/cloudformation` folder.

**Comment by mhuguesaws** [comment]
Thank you for your contribution. See comments

File: fargate/cloudformation/batch-fargate-new-vpc.yaml (hunk 1 above)
Comment (delagoya) [reply fargate/cloudformation/batch-fargate-new-vpc.yaml:1]: No problem. Should I separate the “hello world” job definitions to this project and call out to the core infra as an additional stack with exports?

Comment (mhuguesaws) [reply fargate/cloudformation/batch-fargate-new-vpc.yaml:1]: Please call out the core infra as a step in the process.
You can leave job definition in IaC.

**Comment by mhuguesaws** [comment]
Could you provide an update?

**Comment by delagoya** [comment]
Working on this this week.

**Comment by delagoya** [comment]
Please advise specifically where you would like templates to go. In `templates/ecs-fargate/` or some other directory structure?

**Comment by mhuguesaws** [comment]
just `fargate` at the root directory will be sufficient.

**Comment by delagoya** [comment]
OK, repo should be all good for PR. I just tested manual deployment of resources and running all platform job definitions.

**Review by mhuguesaws — APPROVED** [review]
LGTM

**Pull request merged**